renderer/index.js

```diff
@@ -341,7 +352,20 @@ function jumpToTime (time) {
     state.playing.jumpToTime = time
   }
 }
-
+function setPlaybackRate (direction) {
+  var rate = state.playing.playbackRate
+  if ((direction > 0 && rate >= 1 && rate < 16) || (direction < 0 && rate > -16 && rate <= -1)) {
+    rate *= 2
+  } else if ((direction < 0 && rate > 1 && rate <= 16) || (direction > 0 && rate >= -16 && rate < -1)) {
+    rate /= 2
+  } else if (rate === -1 || rate === 1) {
+    rate *= -1
+  }
```

Comment:
Might want to make this a bit more granular.

The only times I've ever watched videos at a speed other than 1x, it was lecture videos, and I watched them at 1.25x or 1.5x.